action.yml

```diff
@@ -19,5 +19,5 @@
           default: '.'
           required: false
   runs:
-    using: 'node12'
+    using: 'node16'
```

Comment:
aktuelle dependencies z.B. `yaml` verwenden null-coalescing operator, welche in node12 nicht supported sind.

node16 ist aber in allen aktuellen virtual environments vertreten https://github.com/actions/virtual-environments#available-environments